site/docs/v1/tech/apis/_import-webauthn-request-body.adoc

```diff
@@ -44,8 +44,8 @@ The link:/docs/v1/tech/reference/data-types#instants[instant] when the passkey w
 [field]#credentials``[x]``.lastUseInstant# [type]#[Long]# [optional]#Optional# [default]#defaults to now#::
 The link:/docs/v1/tech/reference/data-types#instants[instant] when the passkey was last used.
 
-[field]#credentials``[x]``.name# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**#::
-A unique name meant to disambiguate passkeys with the same [field]#credential.displayName#. If this value is omitted, a string of four alphabetic characters will be generated. The combination of characters will be unique among the passkeys in this bulk import request.
+[field]#credentials``[x]``.name# [type]#[String]# [required]#Required#::
```

Comment:
The field is now required on import because we don't want to look up the login Id for every user on the request.